_DENA/scripts/run_ground_truthing.py

```diff
@@ -49,20 +52,48 @@
 
     # Query flight tracks from days there is NVSPL data for.
     logger.info("Querying tracks...")
-    tracks = query_tracks(engine=engine, start_date=nvspl_dates[0], end_date=nvspl_dates[-1], mask=study_area)
+
+    if args.track_source == 'ADSB':
+        if args.year <= 2019:
```

Comment:
See my comment on helpers.py about using the same function whether we are dealing with early ADSB data or not.